qutip/qip/device/processor.py

```diff
@@ -308,107 +456,100 @@ def read_coeff(self, file_name, inctime=True):
         data = np.loadtxt(file_name, delimiter='\t')
         if not inctime:
             self.coeffs = data.T
+            return self.coeffs
         else:
-            self.tlist = np.hstack([[0], data[:, 0]])
+            tlist = data[:, 0]
+            self.set_all_tlist(tlist)
             self.coeffs = data[:, 1:].T
-        return self.tlist, self.coeffs
+            return self.get_full_tlist, self.coeffs
 
-    def get_unitary_qobjevo(self, args=None):
+    def get_noisy_pulses(self, ind_device_noise=False, ind_drift=False):
         """
-        Create a :class:`qutip.QobjEvo` without any noise that can be given to
-        the QuTiP open system solver.
+        It takes the pulses defined in the `Processor` and
+        add noise according to `Processor.noise`. It does not modify the
+        pulses saved in `Processor.pulses` but return a new list.
+        The length of the new list of noisy pulses might be longer
+        because of drift Hamiltonian and device noise. They will be
+        added to the end of the pulses list.
 
         Parameters
         ----------
-        args: dict, optional
-            Arguments for :class:`qutip.QobjEvo`
+        ind_device_noise: bool, optional
+            If include pulse independent noise such as single qubit
```

Comment:
If "true"? By the way, I would remove `ind` for independent, also in `ind_drift`, or would spell it `independent`.

Reply:
I see, it is confusing. What I mean is `include_device_noise`. What about remove `ind` and make it a  boolean  parameter `device_noise`?